src/test/java/seedu/address/logic/commands/AddNumberOfRoomsCommandTest.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.NewCommandTestUtil.assertCommandSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.RoomBook;
+import seedu.address.model.UserPrefs;
+
+
+
+class AddNumberOfRoomsCommandTest {
+
+    @Test
+    void execute() {
```

Comment:
let's work with the naming together for the test cases next time!

Reply:
Yup, `Address Book Level 3` has a convention for the names of test cases. Maybe we can follow that next time?